src/fx/node.rs

```diff
@@ -0,0 +1,248 @@
+    /// with the kwargs in `HashMap<String, Argument>` in it.
+    /// Otherwise, returns `Err` with a `PyErr` in it.
+    /// The `PyErr` will explain the cause of the failure.
+    pub fn kwargs(&self) -> PyResult<HashMap<String, Argument>> {
+        self.getattr("kwargs")?.extract()
+    }
+
+    /// Retrieve the meta of this `Node`.
+    ///
+    /// If this `Node` has an attribute `meta`, returns `Ok`
+    /// with the meta in `HashMap<String, PyObject>` in it.
+    /// Otherwise, returns `Ok(Default::default())`.
+    /// This never returns `Err`.
+    pub fn meta(&self) -> PyResult<HashMap<String, PyObject>> {
+        if let Ok(meta) = self.getattr("meta") {
+            meta.extract()
+        } else {
+            Ok(Default::default())
+        }
+    }
+
+    fn extract_meta_tensor_meta(&self) -> PyResult<MetaTuple> {
+        Ok(if let Ok(meta) = self.getattr("meta") {
+            let tensor_meta = meta.get_item("tensor_meta").ok().map(TensorMeta::extracts_tensor_meta).transpose().unwrap_or_else(|e| {
+                tracing::debug!("Failed to extract tensor_meta, probably it is gm before 'shape_prop' called., {e:?}");
+                None
+            });
+            let meta = meta.extract()?;
+            (meta, tensor_meta)
+        } else {
+            (Default::default(), Default::default())
+        })
+    }
+
+    fn extract_users(&self) -> PyResult<Vec<String>> {
+        let user_keys = self.getattr("users")?.getattr("keys")?.call0()?;
+        user_keys
+            .iter()?
+            .map(|r| r?.getattr("name")?.extract())
+            .collect()
+    }
+}
+
+impl fmt::Debug for Node {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        #[derive(Debug)]
+        #[allow(unused)]
+        struct Node {
+            origin: PyObject,
+            name: String,
+            op: Op,
+            target: Target,
+            args: Vec<Argument>,
+            kwargs: HashMap<String, Argument>,
+            stack_trace: Option<PyObject>,
+            meta: HashMap<String, PyObject>,
+            tensor_meta: Option<Vec<TensorMeta>>,
+            users: Vec<String>,
+        }
+
+        let (meta, tensor_meta) = self.extract_meta_tensor_meta().map_err(|_| Error)?;
+        write!(
+            f,
+            "{:?}",
+            Node {
+                origin: PyObject::from(self),
+                name: self.name().map_err(|_| Error)?,
+                op: self.op().map_err(|_| Error)?,
+                target: self.target().map_err(|_| Error)?,
+                args: self.args().map_err(|_| Error)?,
+                kwargs: self.kwargs().map_err(|_| Error)?,
+                stack_trace: self
+                    .getattr("stack_trace")
+                    .map_err(|_| Error)?
+                    .extract()
+                    .map_err(|_| Error)?,
+                meta,
+                tensor_meta,
+                users: self.extract_users().map_err(|_| Error)?
+            }
+        )
```

Comment:
`map_err(|_| Error)` 가 반복되는 것을 아래처럼 제거하면 좋겠습니다.
```suggestion
        let node = || -> PyResult<_> {
            Ok(Node {
                origin: PyObject::from(self),
                name: self.name()?,
                op: self.op()?,
                target: self.target()?,
                args: self.args()?,
                kwargs: self.kwargs()?,
                stack_trace: self.getattr("stack_trace")?.extract()?,
                meta,
                tensor_meta,
                users: self.extract_users()?,
            })
        }()
        .map_err(|_| Error)?;
        write!(f, "{node:?}")
```

Reply:
수정했습니다.